qemu/tests/device_option_check.py

```diff
@@ -109,6 +110,17 @@ def run(test, params, env):
                            " Guest output is '%s'" % (params_name,
                                                       output))
 
+        error_context.context("Check serial number length with command %s"
+                              % sg_vpd_cmd, logging.info)
+        sg_vpd_cmd = utils_misc.set_winutils_letter(session, sg_vpd_cmd)
+        _, output = session.cmd_status_output(sg_vpd_cmd)
```

Comment:
Maybe meet some errors here since if the guest os is not windows, the sg_vpd_cmd is `None`.
So session.cmd_status_output(sg_vpd_cmd) -> session.cmd_status_output(None) will raise error such `AttributeError: 'NoneType' object has no attribute 'encode'`

Reply:
Add ' if sg_vpd_cmd:'   to run this setp onlt on windows guest, thanks